packages/contracts/src/core/permission/PermissionConditionBase.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity ^0.8.8;
+
+import {ERC165Upgradeable} from "@openzeppelin/contracts-upgradeable/utils/introspection/ERC165Upgradeable.sol";
+
+import {IPermissionCondition} from "./IPermissionCondition.sol";
+
+/// @title PermissionConditionBase
+/// @author Aragon Association - 2021-2023
+/// @notice An abstract contract to inherit from and to be be implemented to support more customary permissions depending on on- or off-chain state, e.g., by querying token ownershop or a secondary condition, respectively.
+abstract contract PermissionConditionBase is ERC165Upgradeable, IPermissionCondition {
```

Comment:
Why does the condition need to be upgradable? I don't think the framework will reuse deployed conditions from other DAOs so make ig upgradeable is kinda nonsense and a waste of gas

Reply:
> Cloned contracts need to follow similar rules to upgadeable contracts. Mainly that they should have initializer functions instead of constructors. This is why if your contract is going to be cloned using minimal proxies you should be using @openzeppelin/contracts-upgradeable even if your contract is not upgradeable.
Source: https://forum.openzeppelin.com/t/does-a-minimal-proxy-contract-that-creates-clones-need-an-upgradeable-smart-contract/29453/3

We had this discussion with Girogi back then:
https://discord.com/channels/672466989217873929/950422827893420153/1070359525015687269